doc/SETUP.md

```diff
@@ -7,6 +7,8 @@
 
 ### 💻 Load extension into browser
 
+> **NOTE:** by default, the extension built this way will talk to the testnet API (which runs under [api.regtest.getalby.com](https://api.regtest.getalby.com)). The testnet API is **not as stable** as production but you can do transactions without using real money. In case you want to do manual tests against the mainnet API, add the following `ALBY_API_URL` environment variable to your command: `$ ALBY_API_URL="https://api.getalby.com" yarn run dev:your-browser-of-choice`
```

Comment:
```suggestion
> **NOTE:** by default, the extension built this way will talk to the testnet API (which runs under [api.regtest.getalby.com](https://api.regtest.getalby.com)). The testnet API is **not stable** but you can do transactions with testnet sats. In case you want to do run a dev build with the mainnet API, add the following `ALBY_API_URL` environment variable to your command: `$ ALBY_API_URL="https://api.getalby.com" yarn run dev:[chrome|firefox]`
```